src/core.c/CompUnit/Repository/Installation.rakumod

```diff
@@ -231,15 +231,16 @@ sub MAIN(*@, *%) {
         my @*MODULES;
         my $path   = self!writeable-path or die "No writeable path found, $.prefix not writeable";
         my $lock = $.prefix.add('repo.lock').open(:create, :w);
+        LEAVE $lock.close;
         $lock.lock;
+        LEAVE $lock.unlock;
```

Comment:
That's not how `LEAVE` phasers work.  The `LEAVE` phaser will be active for the **entire** block.  So it *will* fire with the `die` in line 232.  And thus cause an execution error because `Any.close` does not work.

Fixed with https://github.com/rakudo/rakudo/commit/7ca96bbd90

Reply:
That commit appears to have broken installation on windows: https://dev.azure.com/Rakudo/rakudo/_build/results?buildId=5249&view=logs&j=70f6b8a1-f3cf-5727-bd79-d08ddb1caf67&t=cd681118-a333-54f7-e221-0bc46b61af45&l=194